apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -500,36 +556,7 @@ async function handleChatCompletionError(
       })
     );
   } else if (error instanceof NetworkError && error.response.status === 403) {
-    const responseBody = await error.response.json();
-    const userType = responseBody?.user_type;
-
-    const userTypeToMessageText: {[key: string]: string} = {
-      teacher: commonI18n.aiChatNotAuthorizedTeacher(),
-      student: commonI18n.aiChatNotAuthorizedStudent(),
-    };
-    const messageText =
-      userTypeToMessageText[userType] ||
-      commonI18n.aiChatNotAuthorizedSignedOut();
-
-    dispatch(
-      addChatEvent({
-        id: getNewMessageId(),
-        text: messageText,
-        notificationType: 'permissionsError',
-        timestamp: Date.now(),
-      })
-    );
-    dispatch(
-      sendAnalytics(
-        EVENTS.SUBMIT_AICHAT_REQUEST_UNAUTHORIZED,
-        {
-          levelPath: window.location.pathname,
-          userType,
-          userMessage: newUserMessage.chatMessageText,
```

Comment:
I dropped this piece of info so I could use this code across the customization and chat unauthorized cases -- I don't think that the user message is a super relevant piece of analytics info in this case (ie, we really just want to know that users are trying to interact with the tool while unauthorized), but I'll confirm with Sam.

Reply:
Confirmed this is ok! [Discussion here](https://codedotorg.slack.com/archives/C06FELVTV0Q/p1729208464827999?thread_ts=1729184315.269629&cid=C06FELVTV0Q)